e2e/multi-svc-app/back-end/Dockerfile

```diff
@@ -30,3 +30,6 @@ RUN chmod +x ./e2e-service
 ENTRYPOINT ["./e2e-service"]
 # The service runs on port 80
 EXPOSE 80
+# The service performs a health check every 50 minutes
+HEALTHCHECK --interval=50m --timeout=3s \
```

Comment:
nit: This is totally fine, but I wonder if we should set numbers more realistic like customers would use? Maybe `--interval=1m`, timeout can remain as is?

Reply:
Okay, makes sense. I purposely made it longer so it wouldn't keep happening while the e2e tests were running, but I guess it doesn't matter. I had started out with 5m because that's the example in the aws docs, but also wanted to use identifiable numbers. 1m works!